db/postgreSQL/seeds/addReviews.js

```diff
@@ -0,0 +1,75 @@
+const generate = require('../../generate.js');
+const fs = require('fs');
+const ObjectsToCsv = require('objects-to-csv');
+
+exports.seed = async (knex) => {
+
+  console.log('Beginning review data seed...');
+  console.time('seed review time');
+  // query to be used to copy CSV file into table
+  const query = `COPY reviews(user_id, book_id, date, review, rating) FROM '/Users/briankwon/Desktop/hr-immersive/SDC/sdc-brian/reviews.csv' DELIMITER ',' CSV HEADER`;
+  let count = 0;
+  // function that deletes csv each iteration to avoid memory error
+  const deleteCSV = () => {
+    fs.unlink('/Users/briankwon/Desktop/hr-immersive/SDC/sdc-brian/reviews.csv', err => {
+      if (err) {console.log(err)};
+    })
+  };
+  // count increment function
+  const inc = () => { count++; };
+  fs.createWriteStream('/Users/briankwon/Desktop/hr-immersive/SDC/sdc-brian/reviews.csv');
+  // truncate will empty the table if filled
+  knex.truncate('reviews');
+  // function that will seed db
+  const seedDB = async() => {
+    while (count < 50) {
+      let fakeReviews = [];
+      for (let i = 0; i < 1000000; i++) {
+        fakeReviews.push(generate.createFakeReview());
+      }
+      await new ObjectsToCsv(fakeReviews).toDisk('/Users/briankwon/Desktop/hr-immersive/SDC/sdc-brian/reviews.csv');
+      await knex.raw(query);
+      await deleteCSV();
```

Comment:
I think we might be throwing `await` out in places where it doesn't do anything.. I would advise looking into how `async`/`await` work with ONLY promises and caution you against using them with callbacks (bc it probably isn't doing what you're hoping it does).

TLDR: `async`/`await` isn't magic and works in a very particular way. If you're not using them in a very particular way, you might be convincing yourself that you're doing something useful (when you're actually not).